tempodb/encoding/vparquet/block_search.go

```diff
@@ -641,8 +416,10 @@ func (r *reportValuesPredicate) KeepColumnChunk(cc parquet.ColumnChunk) bool {
 func (r *reportValuesPredicate) KeepPage(pg parquet.Page) bool {
 	if dict := pg.Dictionary(); dict != nil {
 		for i := 0; i < dict.Len(); i++ {
-			s := dict.Index(int32(i)).String()
-			r.cb(s)
+			v := dict.Index(int32(i))
```

Comment:
i don't recall if i fixed this when we discovered the single page per row group improvement in the other predicates. worth double checking.

Reply:
Good catch, fixed.